### PR TITLE
[WIP] Added content to resources.md

### PR DESCRIPTION
**Pull request by sibasmarak**
### Summary
Addresses #1075 #938 #1169 

This is the brief outline or flow which could help to fix the above-mentioned issues. Do please mention any possible suggestion in the comments below.

### List of changes proposed in this PR (pull-request)
- [x] Designed the content for resources.md
- [x] Modified the content on the basis of comments provided in #1169

### What should a reviewer concentrate their feedback on?

- [ ] The flow, style, and content of the writing
- [ ] Links in the Further Reading section
- [ ] Overall is it ok?

### CODE REVIEW

**Comment by sibasmarak** [comment]
Pinging @malvikasharan to review the content in continuation of #1169. I have closed #1169 as discussed. In this PR I have modified the content as per the resources that you had provided in response to #1169. Please let me know if any modification is further required.

File: book/content/remote_collaboration/resources/resources.md [hunk 1]
```diff
@@ -1,18 +1,78 @@
...
-<!---Add sub-heading--->
+### Stakeholders
+---
+- **Stakeholders** refer to individuals or group of individuals who might affect or be affected by the decisions, actions or policies of an organization.  
+The distribution of the resources takes into account the different stakeholders, and their stakes. 
+The best practice is to distribute the resources in proportionate to the stakes, various stakeholders hold.
+
+- The stakeholders can be categorized mainly into:
+	1. **Internal Stakeholders** which includes the individuals who have collaborated for the project, including the project manager and also the leaders of smaller teams.
+	2. **External Stakeholders** which includes the customers, the testers, partners, vendors.
```
Comment (malvikasharan) [inline book/content/remote_collaboration/resources/resources.md:14]: I personally don't think that this category works for academic or data science project. As mentioned in the article you referred, maybe you can use the same approach to define different stakeholders for the data science, what each of their roles are and what resources they need.

File: book/content/remote_collaboration/resources/resources.md [hunk 2]
```diff
@@ -1,18 +1,78 @@
...
+
+### Management and Sharing of Resources
```
Comment (malvikasharan) [inline book/content/remote_collaboration/resources/resources.md:16]: The content that you have added to section is great but probably doesn't fit well in this subchapter. You can move this entirely to the protocol section of remote collaboration: protocols.md that still needs editing.

File: book/content/remote_collaboration/resources/resources.md (hunk 1 above)
Comment (sibasmarak) [reply book/content/remote_collaboration/resources/resources.md:13]: Ok sure @malvikasharan. I will look into the definitions as per a data science project.

File: book/content/remote_collaboration/resources/resources.md (hunk 2 above)
Comment (sibasmarak) [reply book/content/remote_collaboration/resources/resources.md]: I have moved the content and structured protocols.md a bit. Please check and suggest changes, if any.

**Pull request closed without merging**